README.rst

```diff
@@ -105,7 +105,7 @@ Example Usage
     >>> local.humanize()
     'an hour ago'
 
-    >>> local.humanize(locale='ko_kr')
+    >>> local.humanize(locale='ko-kr')
     '1시간 전'
```

Comment:
I think this is not the output currently, since the last changes to the Korean locale by comfuture.

Reply:
Indeed.

```
>>> import arrow
>>> kr = arrow.utcnow().shift(hours=-1)
>>> kr.humanize(locale='ko_kr')
'한시간 전'
```

Reply:
Fixed